nmdc_runtime/api/endpoints/queries.py

```diff
@@ -29,11 +29,13 @@ def unmongo(d: dict) -> dict:
     return json.loads(bson.json_util.dumps(d))
 
 
-def check_can_delete(user: User):
+def check_can_update_and_delete(user: User):
+    # update and delete queries require same level of permissions
     if not permitted(user.username, "/queries:run(query_cmd:DeleteCommand)"):
         raise HTTPException(
             status_code=status.HTTP_403_FORBIDDEN,
-            detail=(f"Only specific users " "are allowed to issue delete commands.",),
+            detail=(f"Only specific users " 
+                    "are allowed to issue update and delete commands.",),
```

Comment:
I think the space between the two strings is a remnant of a recent change (removing the username list). The strings can be combined into a single one.

```diff
-            detail=(f"Only specific users " 
-                   "are allowed to issue update and delete commands.",),
+             detail=(f"Only specific users are allowed to issue update and delete commands."),
```

Caveat: `black` may still split the single string onto multiple lines due to line length.